### ISSUE TITLE
Sandbox doesn’t support code coverage (tap/nyc/istanbul)

### ISSUE THREAD

**Issue by mikeal** [arc-repos/architect#352]
Tried running my test with coverage and it’s not picking up lambda functions run in the sandbox.

```
tap test/test-put-blocks.js --coverage --nyc-arg=--include=src/http/get-index/index.js

test/test-put-blocks.js ............................... 1/1 3s
total ................................................. 1/1

  1 passing (3s)

  ok
----------|----------|----------|----------|----------|-------------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
----------|----------|----------|----------|----------|-------------------|
All files |        0 |        0 |        0 |        0 |                   |
----------|----------|----------|----------|----------|-------------------|
```
Not sure what is causing this, are the sandboxes run in subprocesses?

**Comment by ryanblock** [comment]
Yeah, we sorta emulate Lambda, each invitation kicks a subprocess that terminates upon completion (or timeout).

**Comment by brianleroux** [comment]
Wondering if subprocess isn't seen by these tools. Interesting problem!

**Comment by brianleroux** [comment]
Here's where we run em https://github.com/arc-repos/architect/blob/master/src/sandbox/http/invoke-lambda.js#L59

Apparently in this file we also follow standardjs formatting now too.  😹🤷‍♂️

**Comment by mikeal** [comment]
nyc supports coverage across subprocess boundaries
https://github.com/istanbuljs/nyc/blob/c88a852cbcdcf38633684dc8f334d25b9a76f17b/README.md

Is there something particular about the way you are launching these?

**Comment by brianleroux** [comment]
looks like it only grabs files directly `require`d so we have to tell it where the src is: https://github.com/brianleroux/arc-example-nyc/blob/master/package.json#L13-L18

gonna leave this open so I can update our docs…this should be in the "working local" tutorial

**Comment by mikeal** [comment]
I don't think that is the issue. If you look at the tap call I'm doing at
the top of the issue I'm explicitly including the lambda's index.js in
nyc's coverage and it still doesn't work.

**Comment by ryanblock** [comment]
TODO: add this to docs! → @brianleroux
